automation_tools/satellite6/upgrade/tasks.py

```diff
@@ -376,35 +377,32 @@ def sync_capsule_repos_to_upgrade(capsules):
 
     Personal Upgrade Env Vars:
 
-    CAPSULE_SUBSCRIPTION
+    CAPSULE_AK
         List of cv_name, environment, ak_name attached to subscription of
         capsule in defined sequence
 
     Rhevm upgrade Env Vars:
 
-    RHEV_CAPSULE_CV
-        The CV name used in capsule subscription
-    RHEV_CAPSULE_ENVIRONMENT
-        The environment name used in capsule subscription
     RHEV_CAPSULE_AK
         The AK name used in capsule subscription
     """
     capsule_repo = os.environ.get('CAPSULE_URL')
     from_version = os.environ.get('FROM_VERSION')
     to_version = os.environ.get('TO_VERSION')
     os_ver = os.environ.get('OS')[-1]
-    cv_name, env_name, ak_name = [
-        os.environ.get(env_var)
-        for env_var in (
-            'RHEV_CAPSULE_CV', 'RHEV_CAPSULE_ENVIRONMENT', 'RHEV_CAPSULE_AK')
-    ]
-    details = os.environ.get('CAPSULE_SUBSCRIPTION')
-    if details is not None:
-        cv_name, env_name, ak_name = [
-            item.strip() for item in details.split(',')]
-    elif not all([cv_name, env_name, ak_name]):
-        print('Error! The CV, Env and AK details are not provided for Capsule'
-              'upgrade!')
+    rhev_ak = os.environ.get('RHEV_CAPSULE_AK')
+    user_ak = os.environ.get('CAPSULE_AK')
+    # Set hammer configuration
+    set_hammer_config()
+    if user_ak is not None:
```

Comment:
I wanted to avoid the repetative code but I was not aware of os.environ.get() has such option. Thanks.